v03_pipeline/lib/tasks/write_family_table.py

```diff
@@ -71,23 +71,21 @@ def requires(self) -> luigi.Task:
     def create_table(self) -> hl.Table:
         pedigree_ht = import_pedigree(self.project_pedigree_path)
         callset_mt = hl.read_matrix_table(self.input().path)
-        callset_family_guids = set(callset_mt.family_guids.collect()[0])
-        if self.family_guid not in callset_family_guids:
-            msg = f'Family: {self.family_guid} was not complete in this callset'
-            raise ValueError(msg)
-        sample_subset_ht = samples_to_include(
-            pedigree_ht,
+        families = parse_pedigree_ht_to_families(pedigree_ht)
```

Comment:
it feels like we would want to do this once, not parse the full set of pedigrees once per family